community/consistency-check/src/test/java/org/neo4j/consistency/checking/GraphStoreFixture.java

```diff
@@ -121,7 +121,10 @@ public TransactionRepresentation representation( IdGenerator idGenerator, int ma
                                                          long lastCommittedTx, CountsTracker counts )
         {
             TransactionWriter writer = new TransactionWriter();
-            transactionData( new TransactionDataBuilder( writer, counts ), idGenerator );
+            try ( CountsAccessor.Updater updater = counts.apply( lastCommittedTx + 1 ).get() )
+            {
+                transactionData( new TransactionDataBuilder( writer, updater ), idGenerator );
+            }
```

Comment:
The `GraphStoreFixture` doesn't apply transactions the "normal" way, so we are having to fabricate what the next transaction id is going to be...